third_party/blink/renderer/modules/cobalt/h5vcc_system/h_5_vcc_system.idl

```diff
@@ -12,15 +12,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+callback Callback = void();
+
+dictionary ConcealOrStopCallbacks {
+  Callback beforeConceal;
+  Callback beforeStop;
+};
+
 // This interface mirrors the H5vccSystem Web IDL defined in Cobalt LTS 25:
 // https://github.com/youtube/cobalt/blob/25.lts.stable/cobalt/h5vcc/h5vcc_system.idl
 [
-    Exposed=Window,
-    SecureContext
+  Exposed=Window,
+  SecureContext
 ]
 interface H5vccSystem {
-    [CallWith=ScriptState, RaisesException]
-    Promise<DOMString> getAdvertisingId();
-    [CallWith=ScriptState, RaisesException]
-    Promise<boolean> getLimitAdTracking();
+  [CallWith=ScriptState, RaisesException]
+  Promise<DOMString> getAdvertisingId();
+  [CallWith=ScriptState, RaisesException]
+  Promise<boolean> getLimitAdTracking();
+  [CallWith=ScriptState, RaisesException]
+  Promise<boolean> concealOrStop(optional ConcealOrStopCallbacks callbacks);
```

Comment:
Could you share what the final interface should look like here? I.e. would we have `userOnExitStrategy`/`getUserOnExitStrategy`?